TECHNICAL_DETAILS.md

```diff
@@ -1,5 +1,29 @@
 # Technical Details #
 
+## Summary of supported features & options ##
+
+| Option  | Status | Override via ASG conf |
+| ------- | :----: | :-------------------: |
+|Run on multiple regions | :white_check_mark: | :heavy_minus_sign: |
+|Keep a fixed minimum percentage of on-demand | :white_check_mark: | :white_check_mark: |
+|Keep a fixed minimum number of on-demand | :white_check_mark: | :white_check_mark: |
+|Can bid at a certain percentage of the on-demand price | :white_check_mark: | :white_check_mark: |
+|Can bid at a higher spot's price | :white_check_mark: | :white_check_mark: |
+|Whitelisting of certain instance types | :x: | :heavy_minus_sign: |
+|Blacklisting of certain instance types | :white_check_mark: | :x: |
+
+| Feature | Status |
+| ------- | :----: |
+|Install via Cloudformation | :white_check_mark: |
+|Install via Terraform | :white_check_mark: |
+|Works with code deploy | :white_check_mark: |
+|Works with beanstalk | :white_check_mark: |
```

Comment:
I'd like to have some lines added about ECS, Kubernetes, Rancher, Mesos, Nomad, etc.(with links to docs and tools that allow draining the workload)
- Also DefaultVPC, EC2Classic and VPC.